### PR TITLE
feat: Add `createdAt` key to json in registry

### PR DESCRIPTION
**Pull request by HazAT**
We want to use the data from our release registry more, one info that is missing today is the timestamp of when something was released.
Yes, in theory, it exists in git, but just for convenience, we want to add a key `createdAt` into every JSON that the Registry target writes for easier access later.

### CODE REVIEW

File: src/targets/registry.ts [hunk 1]
```diff
@@ -362,7 +362,11 @@ export class RegistryTarget extends BaseTarget {
       );
     }
     // Update the manifest
-    const updatedManifest = { ...packageManifest, version };
+    const updatedManifest: { [key: string]: any } = {
+      ...packageManifest,
+      version,
+    };
+    updatedManifest.createdAt = new Date().toISOString();
```
Comment (Lms24) [inline src/targets/registry.ts:369]: Just to confirm: the manifest is what ends up [here](https://github.com/getsentry/sentry-release-registry/blob/master/packages/npm/%40sentry/browser/7.109.0.json), correct? If yes, then I think adding a new field should be safe

Comment (HazAT) [reply src/targets/registry.ts:369]: Yes

**Review by Lms24 — APPROVED** [review]

**Review by Lms24 — APPROVED** [review]
thx!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.18.0
+
+- feat: Registry target now adds key `createdAt` with current timestamp to json file
+
```
Comment (asottile-sentry) [inline CHANGELOG.md:6]: craft will auto generate this based on your PR commit message so this shoudl be left out

File: CHANGELOG.md [hunk 3]
```diff
@@ -108,7 +112,7 @@
 
 ### Various fixes & improvements
 
-- ref: Pin cocoapods version  (#496) by @brustolin
+- ref: Pin cocoapods version (#496) by @brustolin
```
Comment (asottile-sentry) [inline CHANGELOG.md:115]: please revert the other (unrelated, cosmetic) edits to this file -- please make these in a separate PR if you want

File: src/targets/registry.ts [hunk 4]
```diff
@@ -362,7 +362,15 @@ export class RegistryTarget extends BaseTarget {
...
     // Update the manifest
-    const updatedManifest = { ...packageManifest, version };
+    const updatedManifest: {
+      version: string;
+      createdAt: string;
+      [key: string]: any;
+    } = {
+      ...packageManifest,
+      version,
+      createdAt: new Date().toISOString(),
```
Comment (asottile-sentry) [inline src/targets/registry.ts:372]: is there a plan to backfill this data?

any test that needs updating?

Comment (HazAT) [reply src/targets/registry.ts:372]: Added a simple unit test to check if property is there
here is the PR to backfill the data for existing files
https://github.com/getsentry/sentry-release-registry/pull/148

**Review by asottile-sentry — APPROVED** [review]
